magento/pwa-studio
# [Chore] Remove experimental-worker flag for performance.

## Description

Removing the `experimental-worker` flag from jest scripts for performance reasons. Also, it is not needed anymore because we are mocking `MessageChannel` for service worker tests.

`experimental-worker` eats up a lot of memory as shown in the below screen grabs and occasionally jest runs out of memory.

**Jest Heap before (1169 MB):**

`node --experimental-worker --expose-gc ./node_modules/.bin/jest --runInBand --no-cache --logHeapUsage --coverage=false`

![image](https://user-images.githubusercontent.com/35203638/76898755-4ca32500-6864-11ea-981d-25d8c3b3c307.png)

**Jest Heap after changes (361 MB):**

`node --expose-gc ./node_modules/.bin/jest --runInBand --no-cache --logHeapUsage --coverage=false`

![image](https://user-images.githubusercontent.com/35203638/76898163-2b8e0480-6863-11ea-9285-142c59bfd18d.png)

## Related Issue
Closes [PWA-452](https://jira.corp.magento.com/browse/PWA-452)

### Verification Stakeholders
@zetlen 

### Verification Steps
1. Run `node --expose-gc ./node_modules/.bin/jest --runInBand --no-cache --logHeapUsage --coverage=false` to see the heap usage per test. Tests should not fail.

## Checklist
None
